src/fuzz/dummy_fuzz_test.cc

```diff
@@ -0,0 +1,148 @@
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <cstdlib>
+#include <iostream>
+#include <stdexcept>
+#include <filesystem>
+
+#include "frontend/server/server.h"
+#include "google/cloud/spanner/client.h"
+#include "google/cloud/spanner/create_instance_request_builder.h"
+#include "google/cloud/spanner/database_admin_client.h"
+#include "google/cloud/spanner/instance_admin_client.h"
+
+using Server = ::google::spanner::emulator::frontend::Server;
+using ::google::cloud::future;
+using ::google::cloud::StatusOr;
+using ::google::cloud::spanner::DatabaseAdminClient;
+using ::google::cloud::spanner::v0::ConnectionOptions;
+
+#ifdef __OSS_FUZZ__
+#define OVERWRITE 1
+
+bool DoOssFuzzInit() {
+  namespace fs = std::filesystem;
+  fs::path originDir;
+  try {
+    originDir = fs::canonical(fs::read_symlink("/proc/self/exe")).parent_path();
+  } catch (const std::exception& e) {
+    return false;
+  }
+  fs::path tzdataDir = originDir / "data/zoneinfo/";
+  if (setenv("TZDIR", tzdataDir.c_str(), OVERWRITE)) {
+    return false;
+  }
+  return true;
+}
+#endif
+
+const std::string server_address = "localhost:1234";
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size) {
+  #ifdef __OSS_FUZZ__
+    static bool Initialized = DoOssFuzzInit();
+    if (!Initialized) { std::abort(); }
+  #endif
+
+  Server::Options options;
+  options.server_address = server_address;
+  std::unique_ptr<Server> server = Server::Create(options);
+  if (!server) {
+    return EXIT_FAILURE;
+  }
+
+  std::cout << "Server Up, Executing Test Query" << std::endl;
+
+  try {
+    // This is the connection to the emulator.
+    ConnectionOptions emulator_connection;
+    emulator_connection.set_endpoint(server_address)
+        .set_credentials(grpc::InsecureChannelCredentials());
+
+    // First we create an instance to create our database on.
+    google::cloud::spanner::Instance instance("emulator", "emulator");
+    google::cloud::spanner::InstanceAdminClient instance_client(
+        google::cloud::spanner::MakeInstanceAdminConnection(
+            emulator_connection));
+
+    future<StatusOr<google::spanner::admin::instance::v1::Instance>>
+        instance_future = instance_client.CreateInstance(
+            google::cloud::spanner::CreateInstanceRequestBuilder(instance,
+                                                                 "emulator")
+                .SetDisplayName("emulator")
+                .SetNodeCount(1)
+                .SetLabels({{"label-key", "label-value"}})
+                .Build());
```

Comment:
This might be a good place for auto. Also, append the .get() on CreateInstance since you aren't using the future anyway:

auto instance_or = instance_client.CreateInstance(....Build()).get();

The instance_or naming convention is very common in Google.

Also, some using statements might help readability here and elsewhere (it's personal style, whatever you like):
using InstanceBuilder = google::cloud::spanner::CreateInstanceRequestBuilder;